contracts/IdentityManager.sol

```diff
@@ -0,0 +1,219 @@
+pragma solidity 0.4.11;
+import "./Proxy.sol";
+
+
+contract IdentityManager {
```

Comment:
@naterush @oed As I mentioned we need a new contract here. This may actually be a good candidate for inheritance as with the design here the only thing different is the constructor, `onlyAuthorized` and `checkMessageData`. That would also ensure that changes to the normal IdentityManager would get reflected here automatically.